### PR TITLE
add ingested span pricing

### PR DESCRIPTION
**Pull request by andrewardito**
### What does this PR do?
Add Ingestion example to APM pricing doc and add more details around usage metrics for TwL and how to use them.

### Preview

https://docs-staging.datadoghq.com/andrew.ardito/ingestionpricing/account_management/billing/apm_distributed_tracing/
https://docs-staging.datadoghq.com/andrew.ardito/ingestionpricing/tracing/trace_retention_and_ingestion/usage_metrics

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/account_management/billing/pricing.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ Datadog has many pricing plans to fit your needs. For more information, see the
   * On a high watermark plan (HWMP), these hourly measurements are ordered from highest to lowest at the end of the month, and Datadog charges based on the eighth highest measurement.
   * On a hybrid monthly/hourly plan (MHP), Datadog charges your minimum monthly commitment, and for any host hours above that commitment, Datadog charges an hourly rate.
 * An **Indexed Span** is an individual request against an individual service in your stack. Datadog charges based on the total number of spans indexed by [retention filters][3] within Datadog APM.
+* An **Ingested Span** is an individual request against an individual service in your stack.  Datadog charges based on the total number of gigabytes of spans ingested into Datadog APM.
```
Comment (ruthnaebeck) [inline content/en/account_management/billing/pricing.md:29]:
```suggestion
* An **Ingested Span** is an individual request against an individual service in your stack. Datadog charges based on the total number of gigabytes of spans ingested into Datadog APM.
```

File: content/en/account_management/billing/apm_distributed_tracing.md [hunk 2]
```diff
@@ -12,6 +12,7 @@ kind: documentation
...
 | [Indexed span][5] | $1.70 per million Indexed Spans per month | Billed when usage is in excess of Indexed Spans included with every APM host | An Indexed span is an individual request against an individual service in your stack. Datadog charges based on the total number of spans indexed via retention filters or legacy Analyzed Spans to the Datadog APM service at the end of the month. [More information.][5]                                                                                          |
+| [Ingested span][5] | $.10 per GB Ingested Spans per month | Billed when usage is in excess of Ingested Spans included with every APM host | An Ingested span is an individual request against an individual service in your stack. Datadog charges based on the total number of gigabytes of spans ingested to  Datadog at the end of the month. [More information.][5]                                                                                          |
```
Comment (ruthnaebeck) [inline content/en/account_management/billing/apm_distributed_tracing.md:15]:
```suggestion
| [Ingested span][5] | $.10 per GB Ingested Spans per month | Billed when usage is in excess of Ingested Spans included with every APM host | An Ingested span is an individual request against an individual service in your stack. Datadog charges based on the total number of gigabytes of spans ingested to Datadog at the end of the month. [More information.][5]                                                                                          |
```

File: content/en/account_management/billing/apm_distributed_tracing.md [hunk 3]
```diff
@@ -117,6 +119,10 @@ APM is billed on the basis of [hosts][4] deployed with agents sending traces and
 
 As of October 20, 2020, all existing App Analytics filters are automatically transitioned to Retention Filters. You can continue to let the filters remain unchanged or modify them as needed. Transitioned filters are marked with an *i* representing Legacy App Analytics Filters within the [retention filters][3] page.
 
+**8. How do I estimate my Ingested Span volume?**
+
+Datadog provides the metrics `datadog.estimated_usage.apm.ingested_bytes` and `datadog.estimated_usage.apm.ingested_spans` for monitoring ingested span volume.  More information is available in the [Usage Metrics][6] documentation.
```
Comment (ruthnaebeck) [inline content/en/account_management/billing/apm_distributed_tracing.md:124]:
```suggestion
Datadog provides the metrics `datadog.estimated_usage.apm.ingested_bytes` and `datadog.estimated_usage.apm.ingested_spans` for monitoring ingested span volume. More information is available in the [Usage Metrics][6] documentation.
```

**Review by ruthnaebeck — CHANGES_REQUESTED** [review]

File: content/en/tracing/trace_retention_and_ingestion/usage_metrics.md [hunk 4]
```diff
@@ -5,24 +5,40 @@ description: "Learn how to monitor your Tracing without Limits usage."
...
+| Ingestion overview           | Shows a detailed view of ingestion, including bytes per second, total bytes ingested during the time period, and a breakdown of ingestion by `env`.  The same graphs are again available for spans instead of bytes.  **Note:** Billing is based on bytes.|
+| Indexing overview          | Shows a detailed view of indexed spans, including a timeseries of spans indexed per second, total spans indexed over the period, and a breakdown of indexed spans by `env`.  |
+| Per env breakdown |   For each environment, a top list is provided of the ingested and indexed spans from that environment, as well as a top list of what this ratio is per environment.                                                         |
+| Per service breakdown         | For each service, a top list is provided of the ingested and indexed spans from that service, as well as a top list of what this ratio is per service.       |
+| Per env and service combination breakdown         | For each unique combination of `env` and `service`, a top list is provided of the ingested and indexed spans from that environment and service, as well as a top list of what this ratio is per combination. |
+
```
Comment (kayayarai) [inline content/en/tracing/trace_retention_and_ingestion/usage_metrics.md:36]: This seems like we're documenting the UI, which should be self-explanatory, and this table is destined to be wrong almost before it's published. For example, "Summary" is not the heading on that section of the dashboard. I suggest we remove the table and only describe the dashboard generally, not particularly. Can say things like "Billing is based on bytes"

**Review by kayayarai — CHANGES_REQUESTED** [review]

File: content/en/account_management/billing/pricing.md [hunk 5]
```diff
@@ -26,6 +26,9 @@ Datadog has many pricing plans to fit your needs. For more information, see the
   * On a high watermark plan (HWMP), these hourly measurements are ordered from highest to lowest at the end of the month, and Datadog charges based on the eighth highest measurement.
   * On a hybrid monthly/hourly plan (MHP), Datadog charges your minimum monthly commitment, and for any host hours above that commitment, Datadog charges an hourly rate.
 * An **Indexed Span** is an individual request against an individual service in your stack. Datadog charges based on the total number of spans indexed by [retention filters][3] within Datadog APM.
+* An **Ingested Span** is an individual request against an individual service in your stack. Datadog charges based on the total number of gigabytes of spans ingested into Datadog APM.
+
+There are controls in place for both Indexed and Ingested span volumes.  For more information, read the [Trace Rentention and Ingestion Controls][4] documentation.
```
[the remark this answers is not in the document]
Comment (andrewardito) [reply content/en/account_management/billing/pricing.md:31]: Yeah you're completely right. Bad phrasing by me, updated to the suggested.

File: content/en/tracing/trace_retention_and_ingestion/usage_metrics.md (hunk 4 above)
Comment (andrewardito) [reply content/en/tracing/trace_retention_and_ingestion/usage_metrics.md]: Updated - what do you think?

File: content/en/account_management/billing/pricing.md [hunk 6]
```diff
@@ -64,21 +67,22 @@ Datadog has many pricing plans to fit your needs. For more information, see the
   * Each host is allowed up to 4 profiled containers for free. Overage containers will be priced at $2 per container. Note: this allotment is aggregated across all hosts so if you have 4 containers on average across all your hosts, you will not be charged overages on host-by-host basis.
```
Comment (irabinovitch) [inline content/en/account_management/billing/pricing.md:67]: Overage.

Comment (kayayarai) [reply content/en/account_management/billing/pricing.md:67]: Got this in PR 9476

**Review by kayayarai — APPROVED** [review]

**Pull request merged**